astropy/units/format/fits.py

```diff
@@ -55,22 +62,16 @@ def _generate_unit_names():
                     names[key] = special_cases[key]
                 else:
                     names[key] = getattr(u, key)
-        for base in deprecated_bases:
-            for prefix in prefixes:
-                deprecated_names.add(prefix + base)
 
         simple_units = [
             'deg', 'arcmin', 'arcsec', 'mas', 'min', 'h', 'd', 'Ry',
             'solMass', 'u', 'solLum', 'solRad', 'AU', 'lyr', 'count',
             'ct', 'photon', 'ph', 'pixel', 'pix', 'D', 'Sun', 'chan',
-            'bin', 'voxel', 'adu', 'beam'
+            'bin', 'voxel', 'adu', 'beam', 'erg', 'Angstrom', 'angstrom'
         ]
-        deprecated_units = ['erg', 'Angstrom', 'angstrom']
```

Comment:
Can we just leave this in but as empty list? That way, it would be trivial to deprecate other stuff in the future.